backend/pkg/api/connect/integration/api_suite_test.go

```diff
@@ -122,3 +178,64 @@ func (s *APISuite) httpAddress() string {
 	}
 	return "http://" + s.api.Cfg.REST.HTTPListenAddress + ":" + strconv.FormatInt(int64(s.api.Cfg.REST.HTTPListenPort), 10)
 }
+
+func setupNetwork(name string) (*TestNetwork, error) {
+	ctx, cancel := context.WithTimeout(context.Background(), 30*time.Second)
+	defer cancel()
+
+	network, err := testcontainers.GenericNetwork(ctx, testcontainers.GenericNetworkRequest{
+		NetworkRequest: testcontainers.NetworkRequest{
+			Name: name,
+		},
+	})
+	if err != nil {
+		return nil, err
+	}
+	return &TestNetwork{
+		Network: network,
+		Name:    name,
+	}, nil
+}
+
+func runKafkaConnectContainer(network string, bootstrapServers []string) (testcontainers.Container, error) {
+	const testConnectConfig = `key.converter=org.apache.kafka.connect.converters.ByteArrayConverter
+	value.converter=org.apache.kafka.connect.converters.ByteArrayConverter
+	group.id=connectors-cluster
+	offset.storage.topic=_internal_connectors_offsets
+	config.storage.topic=_internal_connectors_configs
+	status.storage.topic=_internal_connectors_status
+	config.storage.replication.factor=-1
+	offset.storage.replication.factor=-1
+	status.storage.replication.factor=-1
+	`
+	const waitTimeout = 3 * time.Minute
+	ctx, cancel := context.WithTimeout(context.Background(), waitTimeout)
+	defer cancel()
+
+	return testcontainers.GenericContainer(ctx, testcontainers.GenericContainerRequest{
+		ContainerRequest: testcontainers.ContainerRequest{
+			Image:        "docker.cloudsmith.io/redpanda/connectors-unsupported/connectors:v1.0.0-3d7ab4d",
```

Comment:
Is this image publicly accessible? Most images for our connectors image used not to be publicly accessible and required cloudsmith auth.

Reply:
This is public but I will use a docker hub image instead